Classes/Domain/Model/Feed.php

```diff
@@ -132,6 +136,14 @@ class Feed extends AbstractEntity
      */
     protected $configuration;
 
+    /**
+     * Fal media items
+     *
+     * @var \TYPO3\CMS\Extbase\Persistence\ObjectStorage<\TYPO3\CMS\Extbase\Domain\Model\FileReference>
+     * @TYPO3\CMS\Extbase\Annotation\ORM\Lazy
+     */
+    protected $falMedia;
```

Comment:
Needs to be initialized like:

```
  public function __construct() {
    // Do not remove the next line: It would break the functionality
    $this->initializeObject();
  }

  /**
   * Initializes all ObjectStorage properties when model is reconstructed from DB (where __construct is not called)
   * Do not modify this method!
   * It will be rewritten on each save in the extension builder
   * You may modify the constructor of this class instead.
   */
  public function initializeObject(): void {
    $this->falMedia = $this->falMedia ?? new ObjectStorage();
  }

```